corehq/messaging/scheduling/models/content.py

```diff
@@ -192,14 +192,6 @@ def get_memoized_app_module_form(self, domain):
 
         return app, module, form, form_requires_input(form)
 
-    def phone_has_opted_out(self, phone_entry_or_number):
-        if isinstance(phone_entry_or_number, PhoneNumber):
-            pb = PhoneBlacklist.get_by_phone_number_or_none(phone_entry_or_number.phone_number)
-        else:
-            pb = PhoneBlacklist.get_by_phone_number_or_none(phone_entry_or_number)
-
-        return pb is not None and not pb.send_sms
```

Comment:
It would be nice if it was documented that `PhoneBlacklist.send_sms` holds the "opt-in/out" state for a phone number. I do not find it to be obvious that that flag is related to opt-in/out now that this function is removed (TBH it was never really obvious).

https://github.com/dimagi/commcare-hq/blob/dc69b6e7ae5dbbd7fa14b6ffaa6d68be999be832/corehq/apps/sms/models.py#L496-L497

Reply:
Where would you recommend documenting that if not where you linked on the model itself?

Reply:
Oh yes, that's what I intended to convey with the link. I think that would be a good place. Sorry for not being more clear.